### PR TITLE
WIP: fixes #524

### LINKED ISSUE #524: Update readme files per feedback
https://github.com/openshiftio/launchpad-addon/issues/46

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

**Comment by gastaldi** [comment]
Can we follow the same pattern for every mission?

**Comment by gastaldi** [comment]
No need to create different Properties files, only the `OSIO-MISSION-README.adoc`

**Comment by rhoads-zach** [comment]
@gastaldi updated the readmes but kept the properties files due to different file paths to edit.

Let me know what you think.

File: docs/topics/readme/osio-README.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
+= ${mission} - ${runtime} Booster
```
Comment (gastaldi) [inline docs/topics/readme/osio-README.adoc]: This file may be deleted

**Comment by Ladicek** [comment]
A lot of the boosters don't work in OSIO (or OSO Starter), but this PR includes READMEs for them as well. I guess that's intentional, but wanted to point out anyway.

**Comment by rhoads-zach** [comment]
Thank you @Ladicek. I added them all for now, but they'll never get used until they are ready in OSIO/OSO.

@gastaldi removed the extra file.

merging.

**Pull request merged**